documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces.adoc

```diff
@@ -1901,8 +1901,7 @@ It is the predefined key “_globalAssetId_” that would refer to the _AssetInf
 |===
 h|Operation Name 4+e|[[PostAllAssetLinksById]]PostAllAssetLinksById
 h|Explanation 4+a|
-Creates new asset identifier key-value-pairs linked to an Asset Administration Shell for discoverable content.
-The existing content might have to be deleted first.
+Creates or replaces all asset identifier key-value-pairs linked to an Asset Administration Shell to edit discoverable content.
```

Comment:
```suggestion
Creates or replaces all asset links associated to the Asset Administration Shell.
```

Reply:
we should not talk about key-value-pairs because they are not. They are triples: name, vale, externalSubjectId